src/nvim/option_vars.h

```diff
@@ -762,6 +762,7 @@ EXTERN int p_wa;                ///< 'writeany'
 EXTERN int p_wb;                ///< 'writebackup'
 EXTERN OptInt p_wd;             ///< 'writedelay'
 EXTERN int p_cdh;               ///< 'cdhome'
+EXTERN char *p_winbd;           ///< 'winborder'
```

Comment:
seems like many of these are alphabetical order, which also helps avoid merge conflicts.

this could go on line